packages/trader/src/Modules/Trading/Components/Form/ContractType/__tests__/contract-type-glossary.spec.tsx

```diff
@@ -17,6 +17,28 @@ describe('<ContractTypeGlossary />', () => {
             screen.getByText(/Contracts will expire at exactly 23:59:59 GMT on your selected expiry date./i)
         ).toBeInTheDocument();
     });
+    it('Ensure multiplier glossary is rendered properly if it is synthetic', () => {
+        render(<ContractTypeGlossary category='multiplier' />);
+
+        expect(screen.getByText('Deal cancellation')).toBeInTheDocument();
+        expect(screen.getByText(/Stop out/i)).toBeInTheDocument();
+    });
+    it('Ensure multiplier glossary is rendered properly if it is fx', () => {
+        render(<ContractTypeGlossary category='multiplier' is_multiplier_fx />);
+
+        expect(screen.queryByText('Deal cancellation')).not.toBeInTheDocument();
+        expect(screen.getByText(/current-tick-execution/i)).toBeInTheDocument();
+        expect(screen.queryByText(/next-tick-execution/i)).not.toBeInTheDocument();
+        expect(screen.getByText(/Stop out/i)).toBeInTheDocument();
+    });
+    it('Ensure multiplier glossary is rendered properly if it is major pairs', () => {
+        render(<ContractTypeGlossary category='multiplier' is_multiplier_fx is_major_pairs />);
+
+        expect(screen.queryByText('Deal cancellation')).not.toBeInTheDocument();
```

Comment:
a nit-pick for maintainability :) it's better to put all reused strings into variables